crates/rust-analyzer/src/cli/flags.rs

```diff
@@ -93,6 +93,10 @@ xflags::xflags! {
             /// and annotations. This is useful for benchmarking the memory usage on a project that has
             /// been worked on for a bit in a longer running session.
             optional --run-all-ide-things
+            /// Run term search
```

Comment:
what does this run it on exactly? Like what expressions etc

Reply:
Currently on all the tail expressions (of functions, block, if statements etc.)

Reply:
I see, could you add that to the description? It's a bit vague what it tries otherwise